src/components/Sum.jsx

```diff
@@ -1 +1,9 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
```

Comment:
you can do return statement right away if you don't need any actions before the return statement

Reply:
and one more thing - we use arrow functions in functional components